app/Http/Controllers/Bible/BiblesController.php

```diff
@@ -92,7 +92,13 @@ public function index()
 
         // removes opus filesets from the request to avoid memory overflows
         list($limit, $is_bibleis_gideons) = forceBibleisGideonsPagination($this->key, $limit);
-        $tag_exclude = isBibleisOrGideon($this->key) ? 'opus' : null;
+        // created to support old bibleis versions
+        $tag_exclude = null;
+        $order_by = 'bibles.id';
+        if (isBibleisOrGideon($this->key)) {
```

Comment:
we want to consider the app version, so this should probably be a call to shouldUseBibleisBackwardCompat instead of isBibleisOrGideon.